From: hololeap <hololeap@protonmail.com>
Signed-off-by: hololeap <hololeap@protonmail.com>

This internal library is only needed for tests (currently disabled) and it
causes an error when building docs:

Running Haddock on library 'spectests-modules' for doctest-parallel-0.2.4
setup: internal error when calculating transitive package dependencies
Debug info: []

Very similar to:
https://github.com/commercialhaskell/stack/issues/5284

diff -urN doctest-parallel-0.2.4/doctest-parallel.cabal doctest-parallel-0.2.4-r1/doctest-parallel.cabal
--- doctest-parallel-0.2.4/doctest-parallel.cabal	2022-08-11 23:35:31.648141338 -0600
+++ doctest-parallel-0.2.4-r1/doctest-parallel.cabal	2022-08-11 23:44:18.178141594 -0600
@@ -119,6 +119,7 @@
 
 
 library spectests-modules
+  buildable: False
   default-language: Haskell2010
   build-depends: base, doctest-parallel, template-haskell
   -- Too many warnings. TODO: fix.
